src/routes.js

```diff
@@ -10,9 +11,18 @@ export const routesNames = {
 };
 
 export const routes = [
-    { path: '/', component: Home },
-    { path: routesNames.adminLogin, component: AdminLogin },
-    { path: routesNames.adminHome, component: AdminHome }
+    {path: '/', component: Home},
+    {path: routesNames.adminLogin, component: AdminLogin},
```

Comment:
To zadziała tylko na próbę wejścia na admin home, a chcemy mieć zabezpieczone wszystkie ścieżki admina, oprócz logowania. Trzeba ustawić https://router.vuejs.org/guide/advanced/navigation-guards.html#global-before-guards i sprawdzać czy ścieżka jest dopuszczalna. Wszystko, co ma admin/(cokolwiek)/.... musi być zakazane